fabtests/man/man7/fabtests.7

```diff
@@ -350,7 +350,7 @@ file.
 .B \f[I]prov_name\f[]
 Identify the provider(s) to test.
 E.g.
-udp, tcp, verbs, ofi_rxm;verbs; ofi_rxd;udp.
+udp, tcp, verbs, ofi_rxm;verbs, ofi_rxd;udp.
```

Comment:
Thanks - for future reference, the actual man pages are automatically generated from the .md files after the changes are merged.

Reply:
Got it, thank you.